OsmAnd/src/net/osmand/plus/views/layers/FavouritesLayer.java

```diff
@@ -111,6 +117,10 @@ public void onDraw(Canvas canvas, RotatedTileBox tileBox, DrawSettings settings)
 		}
 	}
 
+	public void setAndroidAutoFavouritePoints(@Nullable List<FavouritePoint> points) {
```

Comment:
Bad approach. Do not inject AA specific code in layers. For example you may introduce some custom data iterator and assign it to layer(s).